editor/components/inserter/index.js

```diff
@@ -107,10 +107,10 @@ export default compose( [
 		},
 	} ) ),
 	withContext( 'editor' )( ( settings ) => {
-		const { blockTypes, templateLock } = settings;
+		const { allowedBlockTypes, templateLock } = settings;
 
 		return {
-			hasSupportedBlocks: true === blockTypes || ! isEmpty( blockTypes ),
+			hasSupportedBlocks: true === allowedBlockTypes || ! isEmpty( allowedBlockTypes ),
```

Comment:
Since this line is updated. It can be simplified to:

```js
hasSupportedBlocks: ! isEmpty( allowedBlockTypes )
```

Reply:
Very confusingly for us PHP folks, [`_.isEmpty()`](https://lodash.com/docs/#isEmpty) doesn't work the same way as [`empty()`](http://php.net/empty):

```
$ node
> var _ = require('lodash');
> _.isEmpty(true)
true
> _.isEmpty(false)
true

```